### DISCUSSION TITLE
Support for $background_edit

### DISCUSSION THREAD

**Discussion by rkratky** [neomutt/neomutt#4145] [Development]
## What would you like to be able to do?

Support for [$background_edit](http://www.mutt.org/doc/manual/#background-edit).

## What needs to change in NeoMutt?

See description in the linked mutt docs.

## Have you tried any alternative methods?

I would gladly, but I don't know how to work around the missing support.

**Comment by flatcap** [comment]
It's on the list.
I merge all of upstream Mutt's changes, but some of them take time.
That feature was only released two weeks ago.

**Comment by rkratky** [comment]
>  That feature was only released two weeks ago.

Oh, I didn't realize that.

Thanks!

**Comment by christo-auer** [comment]
Any news on this front?

Sometimes, while composing a mail in vim, I need to browse mails in NeoMutt. The
current process is: exit vim, save as draft, browse mails (with copy/paste if
needed), re-open remembered message. The bg-feature would be really helpful to
improve this process.

Thanks!

**Comment by aurelf** [comment]
This would be a great feature to have. Any plans for integrating it ?

**Comment by flatcap** [comment]
It's on the (very long) todo list, but sorry, no progress yet.

I'm tied up in some big changes, see Index v2 #3796 / branch [[flatcap/views]](https://github.com/neomutt/neomutt/tree/flatcap/views) and we don't have many devs.

As always, I'm **happy to mentor** any **NeoMutt volunteers**, but...

Integrating `background_edit` isn't a simple job.
(NeoMutt's code has diverged a long way since its split with Mutt)

The first task is refactoring the **huge** send-email function.
Any volunteer will have to be prepared to _get their hands dirty_ -- dig into the code and understand it.
